src/storage/irc_catalog.cpp

```diff
@@ -137,13 +137,13 @@ DatabaseSize IRCatalog::GetDatabaseSize(ClientContext &context) {
 
 IRCEndpointBuilder IRCatalog::GetBaseUrl() const {
 	auto base_url = IRCEndpointBuilder();
-	base_url.SetPrefix(prefix);
 	base_url.SetWarehouse(warehouse);
 	base_url.SetVersion(version);
 	base_url.SetHost(host);
 	switch (catalog_type) {
 	case ICEBERG_CATALOG_TYPE::AWS_GLUE:
 	case ICEBERG_CATALOG_TYPE::AWS_S3TABLES: {
+		base_url.SetPrefix(prefix);
```

Comment:
Please remove this.
I think this change was made in an attempt to preserve old behavior, but the `SetPrefix` call here has been removed entirely on main already.

Reply:
Removed